app/helpers/compute_resources_helper.rb

```diff
@@ -44,7 +44,7 @@ def memory_options(max_memory)
   end
 
   def password_placeholder(obj, attr = nil)
-    pass = obj.read_attribute(attr).present? || obj.has_attribute?(:password_hash)
+    pass = obj.read_attribute(attr).present? || obj.has_attribute?(:password_hash) if action_name == "edit"
```

Comment:
What about clone actions?

Seems misplaced to put this in here instead of something closer to the controller action or view, doubly so in the compute_resources_helper.